### PR TITLE
Feature/immediate password error

### PR DESCRIPTION
**Pull request by ElliDy**
Let the user see immediately, when the password not fulfills our requirements.

### CODE REVIEW

**Comment by meedheym** [comment]
Seems to work as intented, but I get 400 response (Bad Request) on every keystroke in password field.

**Comment by ElliDy** [comment]
New proposed solution: 
https://yetu-dev.atlassian.net/wiki/display/APPHOME/Password+validation+and+strength
Feedback is welcomed.

**Comment by larsblumberg** [comment]
Updated PR, please review

**Comment by ElliDy** [comment]
Thanks :+1: I will review

File: app/com/yetu/oauth2provider/views/yetuAuthentication/provider.scala.html [hunk 1]
```diff
@@ -24,33 +24,41 @@
...
                             '_label -> Messages("securesocial.signup.password1"),
-                            'class -> "input-xlarge"
+                            'class -> "normal__input",
+                            'placeholder -> Messages("securesocial.signup.password1")
                         )
 
-                        <div class="form-actions">
-                            <button type="submit" class="btn btn-primary">@Messages("securesocial.login.title")</button>
+                        <div class="login__below-form-left">
+                            <!--Here the stay signed in has to be placed-->
```
Comment (larsblumberg) [inline app/com/yetu/oauth2provider/views/yetuAuthentication/provider.scala.html]: @ElliDy Do we need to do something here? If not, can it be removed?

File: conf/routes [hunk 2]
```diff
@@ -79,7 +79,7 @@ POST           /api/authenticate/:provider          @com.yetu.oauth2provider.con
 # New registration page
 GET            /setup/registration                  @com.yetu.oauth2provider.controllers.setup.SetupController.startSignUp
 POST           /setup/registration                  @com.yetu.oauth2provider.controllers.setup.SetupController.handleStartSignUp
-
+POST           /setup/checkPassword                 @com.yetu.oauth2provider.controllers.setup.SetupController.checkPassword
```
Comment (larsblumberg) [inline conf/routes]: @ElliDy Do we actually need this endpoint?

File: app/com/yetu/oauth2provider/views/yetuAuthentication/provider.scala.html (hunk 1 above)
Comment (ElliDy) [inline app/com/yetu/oauth2provider/views/yetuAuthentication/provider.scala.html]: Later there would be something. So just let it like it is for later work ;)

File: conf/routes (hunk 2 above)
Comment (ElliDy) [inline conf/routes]: No, you can remove it. We do not need it anymore.

Comment (ElliDy) [inline conf/routes]: I´m very happy, that is is done now. Big thanks to @larsblumberg and @ghophp. It looks fine for me. I have not tested it yet, but I will do it on the weekend, but I think:
LGTM!

**Comment by ElliDy** [comment]
Strength does not work anymore for me :( 

screen shot 2015-05-09 at 13 01 49

**Comment by ElliDy** [comment]
When I´m in a field, it should be white, although, when it is already valid.

screen shot 2015-05-09 at 13 03 41

**Comment by ElliDy** [comment]
Feedback/Info box should be aligned with questionmark.
screen shot 2015-05-09 at 13 04 54

**Comment by ElliDy** [comment]
Better layouting for space between lines... no consistent. But this we can do in a new pull request.

screen shot 2015-05-09 at 13 08 59

**Comment by ElliDy** [comment]
After password is changed, the link should not say Ok.
screen shot 2015-05-09 at 13 10 08

**Comment by ElliDy** [comment]
In general it looks good. Small minor issues left to fix ;)

**Comment by ghophp** [comment]
Thank you very much for your review Elisa. We should avoid commit code with problems, because I believe that the PRs revision must be just on code-styling, or general review of the code standards. I mean, there is no obligation for testing if the code committed actually work when revising, so we sorry for that.

Specially for the screen "after password is changed", seems that we forgot to replicate our modifications to that screen. We will work a little more on this. Thanks :+1:

**Pull request merged**